src/LoginPage.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { login } from './api';
+
+function LoginPage() {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const history = useHistory();
+
```

Comment:
Consider adding form validation to check the input email and password before submitting the form.